### DISCUSSION TITLE
Moose verbosity for multiapp initialization and tranfer steps

### DISCUSSION THREAD

**Discussion by helen-brooks** [idaholab/moose#16490] [Q&A General] [answered]
I've noticed that although it is possible to turn off console output for apps and sub apps, MOOSE is still pretty verbose when it comes to multiapps and transfers between them. It seems there is no control to turn this off, as the buffer stream first gets sent to `FEProblemBase::_console` (e.g. line 978 in FEProblemBase.C) whereafter it is forwarded to `Moose::out `via `mooseConsole()`.  Am I correct in this conclusion? Is there a simple way to turn off this output?

**Comment by aeslaughter** [comment]
This is not intentional and is likely a bug. You can file an issue if you would like, but not sure if it will get fixed anytime soon. Is this having a negative impact on your simulations?

**Reply by helen-brooks** [reply]
Not sure yet, as my application is still in development and I haven't done much intensive testing yet; but from experience IO can slow things down so my instinct was to turn it off before moving onto some big runs. If I find it's an issue I can just put in a PR; I'm pretty sure it's just a handful of lines in FEProblemBase which could easily be wrapped in a if statement.

**Reply by aeslaughter** [reply]
If the text is going is being streamed to _console then it shouldn't need an if statement. It should not appear with "console = false".

**Comment by helen-brooks** [comment, marked as answer]
Actually, I realised that I can reduce the output to almost nothing, if in addition to console=false I create my own console output object and simply set output_screen = false (and do this for for both the main and sub-app input files). 
```
[Outputs]
  console = false
  [my_console]
    type = Console
    output_screen = false
  []
[]
```
It seems a little circuitous, but it looks to me like I don't have access to the settings of the console object that is created by default.  The only remaining output I get is warnings, which I wouldn't want to get rid of anyway, and a single printout of the name of my subapp.

**Reply by aeslaughter** [reply]
The options available at the top level are limited since they generally are shared by all objects in the [Outputs] block. It is normal to create sub-blocks like this for fine grained control.

For the snippet above you actually should not need the `console = false`, because if a Console object exists in a sub-block then the default object is not created. https://github.com/idaholab/moose/blob/7fedabd8d6ee91b95dca1533fe02d8da9843a70d/framework/src/actions/CommonOutputAction.C#L268

I am glad you found a work around. Could you please create an issue for the problem you are seeing and if possible include an input file that reproduces the problem. Thanks.
